.github/workflows/workflow.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           pip install -r dev-requirements.txt
       - name: Intall draw.io dependencies
         run: |
-          wget -O drawio.deb https://github.com/jgraph/drawio-desktop/releases/download/v12.6.5/draw.io-amd64-12.6.5.deb
```

Comment:
This update is logically part of the adjustment for tests as this would fail the tests, and the new tests would fail without it. Please merge this commit & the commit for updating the tests.

Correction to my previous commit message suggestion:

```
Update draw.io package version

Also updates the expected dimensions for the tests as <url> was resolved.
```

Reply:
Done :heavy_check_mark: